Scholars Portal Journals.js

```diff
@@ -0,0 +1,197 @@
+{
+	"translatorID": "5ac0fd37-5578-4f82-8340-0e135b6336ee",
+	"label": "Scholars Portal Journals",
+	"creator": "Bartek Kawula",
+	"target": "^https?://journals\\d\\.scholarsportal\\.info/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-02-29 19:31:00"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2016 Bartek Kawula
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) { 
+	var myArticles = doc.getElementById('my-articles');
+	if (myArticles) {
+  		Zotero.monitorDOMChanges(myArticles.parentElement, {attributes: true, attributeFilter: ["style"]});
+	}
+	// see if saved list is toggled open
+	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
+		if (getItems(doc, true)) {
+			return 'multiple'
+		}
+	} else if (url.indexOf("/search?q") != -1 || url.indexOf("/browse/") != -1) {
+		if (getItems(doc, true)) {
+			return 'multiple'
+		}
+	} else if (url.indexOf("/details/") != -1) {
+		return 'journalArticle'
+	} 
+}
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if(type == "multiple") {
+		var list = getItems(doc, url);
+		Zotero.selectItems(list, function(selectedItems) {
+			if(!selectedItems) return true;
+			var articles = [];
+			for(var i in selectedItems) {
+				var article = 'http://'+doc.domain+'/ris?uri='+i
```

Comment:
I wasn't part of the original discussion, but is there no better way to do this? Zotero will recognize relative URLs, so `'/ris?uri=' + i` should really be enough and e.g. prevent problems with https vs. http